tracks/python/exercises/isbn-verifier/mentoring.md

```diff
@@ -101,12 +101,13 @@ for position, value in enumerate(isbn):
         try:
             total += int(value) * (10 - position)
         except ValueError:
-	    return False
```

Comment:
Is this meant to be in this PR? It looks like there's no code in the `except` path.

Reply:
Looks like one more indentation level is required.

Reply:
There was a tab symbol and it seems visually it was okay, I've swapped it to 4 spaces but it wasn't enough.